code/2/2.02.constexpr.cpp

```diff
@@ -17,30 +16,31 @@ constexpr int len_foo_constexpr() {
     return 5;
 }
 
-// error in c++11
-// constexpr int fibonacci(const int n) {
-//     if(n == 1) return 1;
-//     if(n == 2) return 1;
-//     return fibonacci(n-1) + fibonacci(n-2);
-// }
-
-// ok
+// error in C++11 due to more than one return statement, okay C++14 and above
```

Comment:
Please improve the description here more sophisticated, for instance, I would say:
The following produces an error in C++11 since multiple return statement exists. However, it is acceptable in C++14 and its successors.